source/scripts/app.js

```diff
@@ -641,14 +641,23 @@ function populateRecipePage(recipeObj, fromSpoonacular) {
   }
 
   for (let i = 0; i < recipeObj.ingredients.length; ++i) {
-    let ingredient = document.createElement("li");
-    ingredient.classList.add("ingredient-item");
-    ingredient.textContent = recipeObj.ingredients[i];
+    let item = document.createElement("div");
+    item.classList.add("ingredient-item");
+
+    let ingredientLabel = document.createElement("label");
+    let ingredientCheckbox = document.createElement("input");
+    ingredientCheckbox.type = "checkbox";
+    ingredientCheckbox.classList.add("ingredient-checkbox");
+    ingredientLabel.appendChild(ingredientCheckbox);
```

Comment:
Is this line necessary, I commented it out when testing and it doesn't seem like it's required.
```suggestion
```

Reply:
Ah, I think that came from one attempt I had that I forgot to delete